startup.py

```diff
@@ -19,12 +27,11 @@
     speedrun_api_key = os.getenv("SPEEDRUN_API_KEY")
     google_api_key = os.getenv("GOOGLE_API_KEY")
 
-    bot = Bot(command_prefix='!')
+    intents = Intents(messages=True, message_content=True, guilds=True)
```

Comment:
These are unbelievably important and there is a massive list of them to keep in mind whenever you add new features. I was tearing my hair out for a while not realizing that you can't do anything with roles if you don't have guilds.

Anyway, "messages" lets us react to messages being sent, "message_content" lets us read messages (to get the commands out) and "guilds" let's us get role access.